posts/admiral/valuelevel/derive_param_computed.qmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Derive a new parameter computed from the value of other parameters"
+author:
+  - name: Kangjie Zhang
+
+description: "Use `admiral::derive_param_computed()` like a calculator to derive new parameters/rows based on existing ones"
+date: "2023-05-03"
+categories: [R, admiral, Valuelevel]
+image: "derive_param_computed.jpg"
+---
+
+<!--------------- my typical setup ----------------->
+
+```{r setup, include=FALSE}
+set.seed(8)
+wide <- 136
+narrow <- 76
+options(width = narrow)
+```
+
+<!--------------- post begins here ----------------->
+
+## Introduction
+
+When creating ADaM Basic Data Structure (BDS) datasets, we often encounter deriving a new parameter based on the analysis values (e.g., AVAL) of other parameters.
+
+The `admiral` function `derive_param_computed()` works like a calculator to derive new records without worrying about merging and combining datasets, all you need is a derivation formula, which also improves the readability of the code. 
```

Comment:
Can these be made into links: admiral and derive_param_computed()

Reply:
links are added